enterprise/backend/src/metabase_enterprise/sso/api/sso.clj

```diff
@@ -53,4 +56,17 @@
       (log/error e (trs "Error logging in"))
       (sso-error-page e))))
 
+(api/defendpoint PUT "/saml/settings"
+  "Update SAML settings. You must be a superuser or have settings permissions to call this API."
+  [:as {{:keys [saml-enabled] :as new-settings} :body}]
+  (validation/check-has-application-permission :setting)
+  (try
+   (let [settings (dissoc new-settings :saml-enabled)]
+     (db/transaction
+      (setting/set-many! settings)
+      (setting/set-value-of-type! :boolean :saml-enabled saml-enabled)))
```

Comment:
Questions

1. Why does this need to be set separately from everything else... does `set-many!` here?
2. Why are we using `set-value-of-type!` as opposed to just using `(saml-enabled! saml-enabled)` to set it? You should generally only be using `set-value-of-type!` inside of custom setters -- you should be using the setter itself everywhere else